app/src/main/java/vn/framgia/vhlee/minicontacts/ContactAdapter.java

```diff
@@ -65,5 +79,27 @@ public void bindData(Contact contact) {
                 mFavourite.setImageResource(R.drawable.ic_favourite);
             }
         }
+
+        public void onItemClick(final int position) {
+            itemView.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    mContactListener.onContactClick(mContacts.get(position));
+                }
+            });
+            mCall.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    mContactListener.onCallClick(position);
+                }
+            });
+
+            mFavourite.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    mContactListener.onFavouriteClick(position);
+                }
+            });
+        }
```

Comment:
Không cần thiết viết ra phương thức này đâu, 
Cứ viết trong constructor của ItemViewHolder
VD: itemView.setOnClickListener(this), rồi implement onClick ra ngoài để xử lý.